supportedBackends/google/batch/src/main/scala/cromwell/backend/google/batch/api/request/BatchRequestExecutor.scala

```diff
@@ -176,8 +176,15 @@ object BatchRequestExecutor {
     }
 
     private def getEventList(events: List[StatusEvent]): List[ExecutionEvent] = {
-      val startedRegex = ".*SCHEDULED to RUNNING.*".r
-      val endedRegex = ".*RUNNING to.*".r // can be SUCCEEDED or FAILED
+      // on Batch, when job transitions to SCHEDULED state it indicates that the VM is being initialized. Users are billed for this
+      // startup time. Hence, the 'vmStartTime' corresponds to when the job enters the SCHEDULED state.
+      val startedRegex = ".*QUEUED to SCHEDULED.*".r
```

Comment:
Any chance we might be transitioning to scheduled from a state other than QUEUED?

Reply:
Do we want the start regex to be `".*to SCHEDULED.*"` to be safe?